File: src/main/java/org/cimsbioko/utilities/DateUtils.java
package org.cimsbioko.utilities;

import org.cimsbioko.navconfig.UsedByJSConfig;

import java.text.SimpleDateFormat;
import java.util.Date;

public class DateUtils {

    @UsedByJSConfig
    public static String formatTime(Date date) {
        return new SimpleDateFormat("yyyy-MM-dd HH:mm:ss").format(date);
    }

    @UsedByJSConfig
    public static String formatDate(Date date) {
        return new SimpleDateFormat("yyyy-MM-dd").format(date);
    }

}

File: src/main/kotlin/org/cimsbioko/utilities/DateUtils.kt
package org.cimsbioko.utilities

import android.annotation.SuppressLint
import org.cimsbioko.navconfig.UsedByJSConfig
import java.text.SimpleDateFormat
import java.util.*

@SuppressLint("SimpleDateFormat")
object DateUtils {

    @UsedByJSConfig
    fun formatTime(date: Date?): String = date?.let { SimpleDateFormat("yyyy-MM-dd HH:mm:ss").format(it) } ?: ""

    @UsedByJSConfig
    fun formatDate(date: Date?): String = date?.let { SimpleDateFormat("yyyy-MM-dd").format(date) } ?: ""

}